bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```
@@ -122,7 +122,7 @@ spec:
               value: {{ .Values.persistence.mountPath | quote }}
             - name: PGDATA
               value: {{ printf "%s/%s" .Values.persistence.mountPath "data" | quote }}
-            {{- if and .Values.postgresql.postgresPassword (not (eq .Values.postgresql.username "postgres")) }}
+            {{- if and (or .Values.postgresql.existingSecret .Values.postgresql.postgresPassword) (not (eq .Values.postgresql.username "postgres")) }}
```

Better.. But we still have a problem.. This is not considered "global" values.
```suggestion
            {{- if and (or (include "postgresql-ha.postgresqlCreateSecret" .) (include "postgresql-ha.postgresqlPostgresPassword" .)) (not (eq (include "postgresql-ha.postgresqlUsername" .) "postgres")) }}
```